src/core/lv_obj_pos.h

```diff
@@ -340,14 +340,25 @@ void lv_obj_move_children_by(lv_obj_t * obj, int32_t x_diff, int32_t y_diff, boo
  */
 void lv_obj_transform_point(const lv_obj_t * obj, lv_point_t * p, bool recursive, bool inv);
 
+/**
+ * Transform an array of points using the angle and zoom style properties of an object
+ * @param obj           pointer to an object whose style properties should be used
+ * @param points        the array of points to transform, the result will be written back here too
+ * @param count         number of points in the array
+ * @param recursive     consider the transformation properties of the parents too
+ * @param inverse       do the inverse of the transformation (-angle and 1/zoom)
```

Comment:
How about use a single `flag` parameter defined as "`LV_OBJ_TRANSFORM_POINT_INVERSE | LV_OBJ_TRANSFORM_POINT_RECURSIVE`"?

It could improve code readability considering below usage.
`lv_obj_transform_point_array(obj, points, n, true, true)`